zhaquirks/nodon/pilot_wire.py

```diff
@@ -0,0 +1,134 @@
+"""NodOn pilot wire heating module."""
+
+from zigpy.profiles import zgp, zha
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl import foundation
+from zigpy.zcl.clusters.general import (
+    Basic,
+    GreenPowerProxy,
+    Groups,
+    Identify,
+    OnOff,
+    Ota,
+    Scenes,
+)
+from zigpy.zcl.clusters.lightlink import LightLink
+from zigpy.zcl.clusters.smartenergy import Metering
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+
+class NodOnPilotWireManufacturerSpecificCluster(CustomCluster):
+    """NodOn manufacturer specific cluster to set Pilot Wire mode."""
+
+    # Codes taken from
+    # https://github.com/Koenkk/zigbee-herdsman-converters/blob/0f4833340a20db3dae625a61c41d9be0a6f952be/src/converters/fromZigbee.ts#L5285.
+    class PilotWireMode(t.enum8):
+        """Pilot Wide Modes available in the device."""
+
+        Off = 0x00
+        Comfort = 0x01
+        Eco = 0x02
+        FrostProtection = 0x03
+        ComfortMinus1 = 0x04
+        ComfortMinus2 = 0x05
+
+    name: str = "NodOnPilotWireManufacturerSpecificCluster"
+    cluster_id: t.uint16_t = 0xFC00
+    ep_attribute: str = "nodon_pilot_wire_manufacturer_specific"
+
+    attributes = {
+        0x0000: ("mode", PilotWireMode, True),
+    }
+
+    server_commands = {
+        0x0000: foundation.ZCLCommandDef(
+            "setMode",
+            {"mode": PilotWireMode},
+            False,
+            is_manufacturer_specific=True,
+        ),
+    }
```

Comment:
These should be converted to the new-style zigpy `AttributeDefs`, see an example here:
https://github.com/zigpy/zha-device-handlers/blob/18c2292c14c32ea016d883e8228340d46b2e8930/zhaquirks/legrand/cable_outlet.py#L51-L58

The same is also true for server commands, example here:
https://github.com/zigpy/zha-device-handlers/blob/18c2292c14c32ea016d883e8228340d46b2e8930/zhaquirks/legrand/cable_outlet.py#L89-L97

Reply:
done